### PR TITLE
Remove K8S Maintain hardcode. && Use ip as the default nodename of the host.

### PR DESCRIPTION
**Pull request by ydye**

### CODE REVIEW

File: pai-management/node-list-example.yaml [hunk 1]
```diff
@@ -16,20 +16,25 @@
 # OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
 
 
-machinelist:
+machine-list:
```
Comment (hwuu) [inline pai-management/node-list-example.yaml]: Where is this file used?

File: pai-management/paictl.py [hunk 2]
```diff
@@ -269,12 +268,20 @@ def pai_build():
...
+    logger.error("Add New Machine Node into cluster     :  paictl.py machine add -p /path/to/configuration/ -l /path/to/nodelist.yaml")
+    logger.error("Remove Machine Node from cluster      :  paictl.py machine add -p /path/to/configuration/ -l /path/to/nodelist.yaml")
+    logger.error("Repair Issue Machine Node in cluster  :  paictl.py machine add -p /path/to/configuration/ -l /path/to/nodelist.yaml")
+    logger.error("Repair Issue k8s node in cluster      :  paictl.py machine add -p /path/to/configuration/ -l /path/to/nodelist.yaml")
 
 
 
+def pai_machine():
+
+    None
```
[the remark this answers is not in the document]
Comment (hwuu) [reply pai-management/paictl.py]: I think yundong will implement it soon. Right? @ydye

File: pai-management/paictl.py [hunk 3]
```diff
@@ -269,12 +268,20 @@ def pai_build():
 
 
 
-def pai_control():
+def pai_machine_info():
```
[the remark this answers is not in the document]
Comment (hwuu) [reply pai-management/paictl.py]: @fanyangCS : This one is necessary as soon as pai_machine() function is implemented. I think this PR is only focus on fixing the 'hardcode' issue. The implementation of pai_machine() will be given in another separate PR. Am I right @ydye ?

File: pai-management/paictl.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (wangcan0329) [reply pai-management/paictl.py]: I remember we scheduled pai_machine() implementation to next release.

File: pai-management/node-list-example.yaml (hunk 1 above)
Comment (wangcan0329) [reply pai-management/node-list-example.yaml]: machine-list will be used in pai_machine() related commands, please refer to [paictl readme](https://github.com/Microsoft/pai/blob/management-dev/pai-management/doc/paictl.md)

File: pai-management/paictl.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (ydye) [reply pai-management/paictl.py]: Yes, will implement it soon. But before do it. I need change some configuration's code.

File: pai-management/paictl.py (hunk 3 above)
[the remark this answers is not in the document]
Comment (ydye) [reply pai-management/paictl.py]: @hwuu You are right.

File: pai-management/node-list-example.yaml (hunk 1 above)
Comment (ydye) [reply pai-management/node-list-example.yaml]: I will do some change this week, to make this style compatible with k8sClusterMangement tool

**Pull request merged**